bedrock/firefox/templates/firefox/features/pdf-editor-fr.html

```diff
@@ -96,10 +95,9 @@ <h2>Modifier un PDF en ligne rapidement et facilement</h2>
 
 <p>Modifier des PDF oui, mais de façon simple et sécurisée !</p>
 
-<hr>
-
-<h2>Vous n'avez pas encore Firefox ?</h2>
-<p><a href="{{ url('firefox.features.index') }}">En savoir plus</a> sur nos fonctionnalités, ou <a href="">Téléchargez Firefox</a> dès maintenant.</p>
-
+<div class="hide-from-firefox">
+  <h2>Vous n'avez pas encore Firefox ?</h2>
+  <p><a href="{{ url('firefox.features.index') }}" data-cta-link="Learn more about our features">En savoir plus</a> sur nos fonctionnalités, ou <a href="{{ url('firefox.new') }}" data-link-text="Download Firefox" data-link-position="content">Téléchargez Firefox</a> dès maintenant.</p>
```

Comment:
**Question:** Does the first anchor tag in this line also need a `data-link-position`?

Reply:
No, there's nothing else to confuse this one with.